pyvista/core/filters/data_set.py

```diff
@@ -2778,16 +2778,11 @@ def _extract_points(_input, points):
             # PolyData with 'largest' mode generates bad output with unreferenced points
             output_needs_fixing = True
 
-        else:
-            # All other extraction modes / cases may generate incorrect scalar arrays
-            # e.g. 'largest' may output scalars with shape that does not match output mesh
-            # e.g. 'seed' method scalars may have one RegionId, yet may contain many
-            # disconnected regions. Therefore, check for correct scalars size
```

Comment:
@tkoyama010 I haven't kept track of all the formatting changes as of lately, but I don't like that this one removed some comments for explaining the `else` case (not only here, as can be seen in the diff). Personally, I also find a nested `else`-`if` tree sometimes more clear than this flattened `elif` result, but that's more a stylistic preference I guess.

Reply:
Good point. I will revert this once and reconsider.